MANIFEST.in

```diff
@@ -1,5 +1,5 @@
-include pytorch_lightning/py.typed  # marker file for PEP 561
-include CHANGELOG.md
```

Comment:
It's a good idea to distribute the CHANGELOG. Why remove it?

Reply:
it is there: https://github.com/Lightning-AI/lightning/blob/55e347ba609df1a1248ef1243ff8cbc09fb97bc8/src/pytorch_lightning/__setup__.py#L56